.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- TODO: Replace the title below -->
+# Your Title
+
+## Description
+
+<!-- TODO: Write a description of the proposed changes -->
+
+### Type of Change
+
+<!-- TODO: Fill in the brackets with an `x` next to all types that apply to the proposed changes -->
+- [ ] Feature
+- [ ] Bug fix
+- [ ] Documentation
+
+### Test Coverage
+
+<!-- TODO: Fill in the test coverage -->
+`0%`
```

Comment:
Hi @ullrichd21! Could you explain the purpose for this section? How should students fill out this section? For instance, should the coverage include both statement and branch coverage? What coverage measuring tool should students use?

Reply:
I didn't think about the coverage requirement in the CI. This is already in the github actions so I think that might cover things. Removing this in the PR template.